tests/common/util.rs

```diff
@@ -1834,6 +1834,14 @@ impl UCommand {
         cmd_result
     }
 
+    #[track_caller]
+    pub fn fails_with_code(&mut self, expected_code: i32) -> CmdResult {
```

Comment:
why this function?
you can do
`.fails().code_is(1)`

Reply:
I find it to be faster to write (it only takes 1 LSP completion, which loads in 3-4 seconds on my machine), and takes 1 line less after formatting.
Considering the fact I had to replicate it around 30 times last evening, I thought it was worth adding, but obviously that's not a strong requirement.

Reply:
why not but i would have preferred to see this in a dedicated PR